SPEC.md

```diff
@@ -85,33 +85,54 @@ This specification outlines the requirements for botbuilder samples.
 ```markdown
 <INSERT AT MOST ONE PARAGRAPH DESCRIPTION OF WHAT THIS SAMPLE DOES> 
 
+# Table of Contents
+<LINKS TO SECTIONS OF THE README>
+
 # Concepts introduced in this sample
 <DESCRIPTION OF THE CONCEPTS>
 
+#Services used in this sample
+<INTRODUCE SERVICES>
 
 # To try this sample
--	<STEPS TO CLONE REPO AND GET SETUP>
+
 ## Prerequisites
--	<REQUIRED TOOLS, VERSIONS>
--	<STEPS TO GET SET UP WITH THE SAMPLE. E.g. RUN AN INCLUDED SCRIPT OR MANUALLY DO SOMETHING ETC>
+
+###	Required Tools
+-   <Required TOOLS WITH MINIMUM VERSION NUMBERS>
+
+### Clone the repo
+-	<STEPS TO CLONE REPO AND GET SETUP>
 
 NOTE: <ANY NOTES ABOUT THE PREREQUISITES OR ALTERNATE THINGS TO CONSIDER TO GET SET UP>
 
-## Visual studio
+## Ways to get started with Bot Framework Samples
```

Comment:
I'd suggest a ##Prerequisites section, followed by ##Configure services section and then a ##Run the sample which can have VS and VS code as sub sections. 

So something like 
# To try this sample
## Prerequisites
### Service accounts and subscriptions
### Required tools
## Configure services
### Using CLI tools
### Using portal
## Run the sample (depending on the sample itself we will only show one of these - C# samples will have VS section and node samples will have VS code).
### Visual studio
### VS code